Makefile

```diff
@@ -17,28 +17,30 @@ restore: restore-in-folder replace-original
 
 AWS_FOLDER ?= "$(HOME)/.aws"
 .PHONY:
-restore-in-folder: validate-input validate-no-previous-restored
+restore-in-folder: validate-prefix validate-no-previous-restored
+	@echo "Validating that PREFIX does not have trailing slash..."
+	$(shell echo $(PREFIX) | grep -q '/$$' && (echo "error: PREFIX should not have trailing slash"; exit 2))
```

Comment:
This repeats what the target `validate-prefix` does no? I guess we don't need it.
```suggestion
```

Reply:
Good catch! Removing it in 3ae1ff96a1b3b406028cff8899b93ae961aeedf3